src/components/others/navigation/index.jsx

```diff
@@ -54,27 +60,22 @@ export default function Navigation() {
 
 function OverlayNav({ isMenuOpen, toggleMenu }) {
   const ethereumContext = useContext(EthereumContext);
+
+  const links = [{name:"Front Page", to: `${ethereumContext?.chainId || Object.keys(networkMap)[0]}/`},
+    {name:"Report", to: `${ethereumContext?.chainId || Object.keys(networkMap)[0]}/report`},
+    {name:"Account", to: `${ethereumContext?.chainId || Object.keys(networkMap)[0]}/account`},
```

Comment:
Account Route doesn't work as expected. At first glance, this line seems to be trouble. `to` field is set to the second operand when ethereumContext is undefined and is never updated afterward.